autoload/db_ui/schemas.vim

```diff
@@ -106,5 +122,12 @@ endfunction
 
 function! db_ui#schemas#query(db, query) abort
   let base_query = db#adapter#dispatch(a:db.conn, 'interactive')
-  return map(systemlist(printf('%s %s', base_query, a:query)), 'substitute(v:val, "\r$", "", "")')
+  return map(
+  \   systemlist(
+  \     a:db.scheme ==# 'oracle' ?
```

Comment:
Because `sqlplus` requires piping queries, __e.g.__:

```sh
mysql -e '<query>'
echo '<query>' | sqlplus
```

…there must be logic to reverse them.